### PR TITLE
Update Julia version in CI to 1.7

### PR DESCRIPTION
**Pull request by uncomfyhalomacro**

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Comment by uncomfyhalomacro** [comment]
> LGTM rocket Thank you for your contribution! smile+1

Btw can u merge this after the tests? hoping this will get merged since 1.7 is the new stable release

**Pull request merged**
